deploy/apex/overlays/cockroach/kustomization.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - ../dev
+  - cockroach.yaml
+  - secrets.yaml
+
+namespace: apex
+patchesJson6902:
+  - target:
```

Comment:
If it's possible, I would set that environment variable from a configmap (or secret) and override the configmap value here using:
```yaml
configMapGenerator:
  - name: ipam
     behavior: replace
     literals:
       - enable_ssl=disable
```

This is less likely to fail if we change the ordering or environment variables (i.e someone adds a new one to the top of the list.

Reply:
honestly we also need to be able to pass the DB certs like this too.  The cockroach db operator is not doing the automatic cert magic that the crunchy operator is doing and for prod use we will need to configure client and ca certs the the apiserver and ipam pods.